sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ParallelTransferOptions.java

```diff
@@ -0,0 +1,63 @@
+package com.azure.storage.blob.models;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.common.Constants;
+import com.azure.storage.common.Utility;
+
+public class ParallelTransferOptions {
+    private final ClientLogger logger = new ClientLogger(ParallelTransferOptions.class);
+
+    static final int BLOB_DEFAULT_UPLOAD_BLOCK_SIZE = 4 * Constants.MB;
+    static final int BLOB_MAX_BLOCK_SIZE = 100 * Constants.MB;
+
+    static final int BLOB_DEFAULT_NUMBER_OF_PARALLEL_TRANSFERS = 2;
```

Comment:
If I'm understanding this right, I think this would typically map to the number of buffers that would be used. I think we should make that more explicit as it might surprise someone trying to speed things up and they suddenly run out of memory

Reply:
Make the default value more explicit, like in comments? I was talking to Amanda and she was saying .NET uses 8 as the default.

Reply:
Changed variable name to numBuffers